feeds/fixtures/feeds/rss10.xml

```diff
@@ -12,6 +12,10 @@
   <channel rdf:about="http://meerkat.oreillynet.com/?_fl=rss1.0">
     <title>Meerkat</title>
     <link>http://meerkat.oreillynet.com</link>
+    <cloud>http://meerkat.oreillynet.com</cloud>
+    <image>
+        <url>http://meerkat.oreillynet.com/someimage.jpg</url>
+    </image>
```

Comment:
I don't think `cloud` is in RSS 1.0, and `image` should look more like `<image rdf:resource="{image_uri}" />`. Are these RSS 2.0 elements maybe? Should we move them to that fixture?